app/com/arpnetworking/metrics/portal/alerts/impl/DailyPartitionCreator.java

```diff
@@ -319,6 +316,8 @@ protected void execute(
         // while SQLUpdate does not allow for SELECT statements.
         //
         // https://ebean.io/docs/intro/queries/jdbc-query
+        //
+        // TODO(cbriones): Move DB operation off the dispatcher thread pool.
```

Comment:
I didn't do this right away to keep this diff from growing any larger. I'm not really concerned by this operation anyway, since it's extremely infrequent.